change/@fluentui-eslint-plugin-309e14e3-2c53-4422-b823-9a8d71aa4b0a.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "support excluded files from being linted",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "mgodbolt@microsoft.com",
+  "dependentChangeType": "patch"
```

Changes to eslint-plugin need to use `dependentChangeType: none` to avoid unnecessary bumps of every single package.
```suggestion
  "dependentChangeType": "none"
```

---

updated in [73705bf](https://github.com/microsoft/fluentui/pull/20466/commits/73705bfb111be4387b8b34202fdd02db9052848d)